tests/unit/db/redshift/test_redshift_db_driver_format_negotiation.py

```diff
@@ -33,10 +33,10 @@ def test_can_load_this_format(self):
             assert_called_with(mock_source_records_format)
         self.assertEqual(out, self.mock_redshift_loader.can_load_this_format.return_value)
 
-    def test_can_unload_this_format(self):
+    def test_can_unload_format(self):
```

Comment:
To stick with the style of most test names, shouldn't this have `_true` at the end?

Reply:
✔️